### PR TITLE
Added labels instead of protoPayloads

### PR DESCRIPTION
**Pull request by google-nalin**

### CODE REVIEW

File: alerts/google-quotas/all-adjustments-by-quota-adjuster.v1.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
       {
           "displayName":"Log match condition",
           "conditionMatchedLog":{
-              "filter":"log_id(\"cloudaudit.googleapis.com/system_event\")\nprotoPayload.methodName=\"google.cloud.quotaadjuster.v1main.QuotaAdjusterService.AutoAdjustQuota\"\nprotoPayload.metadeta.service=\"${SERVICE_NAME}\" \nprotoPayload.metadata.state=\"SUCCEEDED\"",
+              "filter":"log_id(\"cloudaudit.googleapis.com/system_event\")\nprotoPayload.methodName=\"google.cloud.quotaadjuster.v1main.QuotaAdjusterService.AutoAdjustQuota\"\nprotoPayload.metadata.state=\"SUCCEEDED\"",
```
Comment (magicyuli) [inline alerts/google-quotas/all-adjustments-by-quota-adjuster.v1.json:12]: as discussed let's use labels instead

File: alerts/google-quotas/quota-adjuster-errors-and-failures.v1.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     {
       "displayName": "Log match condition",
       "conditionMatchedLog": {
-        "filter": "log_id(\"cloudaudit.googleapis.com/system_event\") \nprotoPayload.methodName=\"google.cloud.quotaadjuster.v1main.QuotaAdjusterService.AutoAdjustQuota\"\nprotoPayload.metadeta.service=\"${SERVICE_NAME}\"  \nprotoPayload.metadata.state=\"FAILED\"",
+        "filter": "log_id(\"cloudaudit.googleapis.com/system_event\") \nprotoPayload.methodName=\"google.cloud.quotaadjuster.v1main.QuotaAdjusterService.AutoAdjustQuota\"\nprotoPayload.metadata.state=\"FAILED\"",
```
Comment (magicyuli) [inline alerts/google-quotas/quota-adjuster-errors-and-failures.v1.json:12]: same here

**Review by magicyuli — APPROVED** [review]

**Comment by google-nalin** [comment]
Screenshot of alert templates created 
image

**Comment by google-nalin** [comment]
image

**Review by magicyuli — APPROVED** [review]

**Review by cocosheng — APPROVED** [review]

**Pull request merged**